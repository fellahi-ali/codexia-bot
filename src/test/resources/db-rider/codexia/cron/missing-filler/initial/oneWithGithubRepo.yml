codexia_project:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 1662
        coordinates: "test-project1/test-repo1"
        author: "iakunin"
        deleted: null
        project_created_at: "2020-05-05 10:29:21"

github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

github_repo_source:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "1662"
