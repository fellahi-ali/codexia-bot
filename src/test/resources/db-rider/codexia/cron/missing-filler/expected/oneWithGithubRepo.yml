codexia_project:
    -
        external_id: 1662
        coordinates: "test-project1/test-repo1"
        author: "iakunin"
        deleted: null
        project_created_at: "2020-05-05 10:29:21"

github_repo:
    -
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

github_repo_source:
    -
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "1662"
